File: javascript/1134.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var a = 0;
var g = 0;
var d = 0;
for(let i = 0; i <= lines.length; i++) {
     if(lines[i] == 1) {
          a++
     } else if(lines[i] == 2) {
          g++
     } else if(lines[i] == 3) {
          d++
     } else if(lines[i] == 4) {
          break
     } else {
          
     }
}
console.log("MUITO OBRIGADO");
console.log("Alcool: "+a);
console.log("Gasolina: "+g);
console.log("Diesel: "+d);

File: C/1134.c
#include <stdio.h>

int main(){

    int escolha, alcool = 0, gasolina = 0, diesel = 0;
    scanf("%d", &escolha);
    
    while(escolha != 4) {
        switch(escolha) {
            case 1:
                alcool++;
                break;
            case 2:
                gasolina++;
                break;
            case 3:
                diesel++;
                break;
            default:
                break;
        }
        scanf("%d", &escolha);
    }
    
    printf("MUITO OBRIGADO\n");
    printf("Alcool: %d\n", alcool);
    printf("Gasolina: %d\n", gasolina);
    printf("Diesel: %d\n", diesel);

    return 0;
}